### PR TITLE
Added output of number of tasks to be modified in batch mode

### PR DESCRIPTION
**Pull request by sebu06**
#### Description

Added number of tasks as output for batch operations, see issue #2532.

#### Additional information...

- [X] Have you run the test suite?
Passed:                          2717
Failed:                             0
Unexpected successes:               0
Skipped:                            0
Expected failures:                  4
Runtime:                         0.35 seconds

### CODE REVIEW

File: src/commands/CmdAnnotate.cpp [hunk 1]
```diff
@@ -70,6 +70,9 @@ int CmdAnnotate::execute (std::string&)
   // Accumulated project change notifications.
   std::map <std::string, std::string> projectChanges;
 
+  if(filtered.size() > 1) {
+    std::cout << "This command will alter " << format(filtered.size()) << " tasks." << std::endl;
```
Comment (tbabej) [inline src/commands/CmdAnnotate.cpp:74]: Taskwarrior uses [verbosity tokens](https://taskwarrior.org/docs/verbosity.html) to allow users to control the amount of output and feedback messages the `task` commands are providing.

Hence we should not output this directly to `cout`, but rather make this output appear under the appropriate verbosity token. Looking at the list the `affected` seems to be the right match here (from `man taskrc`):

```
                  affected   Reports 'N tasks affected' and similar
```

There is already a convenience function called `feedback_affected` that does the check for you:

```c++
      feedback_affected ("This command will alter {1} tasks.", count);
```

**Review by tbabej — CHANGES_REQUESTED** [review]

File: src/commands/CmdAnnotate.cpp (hunk 1 above)
Comment (sebu06) [reply src/commands/CmdAnnotate.cpp]: Hi,
thanks - as you've seen I'm new to contributing to taskwarrior and still finding my way through the codebase. I included your request in the next commit. It is very positive for me to see that there are quick responses to pull requests - this definetely increases my motivation to contribute - thank you again for that.

**Review by tbabej — APPROVED** [review]
Looks good to me. Thanks for the PR!

File: src/commands/CmdAnnotate.cpp (hunk 1 above)
Comment (tbabej) [reply src/commands/CmdAnnotate.cpp]: No worries! I'm happy to see new contributors to the project. Feel free to shoot me an email if you have any questions :slightly_smiling_face:

**Pull request merged**